ui/src/pages/Game.js

```diff
@@ -48,27 +66,67 @@ class GamePage extends Component {
         }
       })
     .then((response) => {
-        console.log(response)
-        this.setState({ ready: true });
+      this.saveState(response.data.result[0])
+    })
+    .catch(function (error) {
+        console.log(error);
+    });
+  }
+
+  nextTurn() {
+    const { params: { gameId } } = this.props.match;
+    axios({
+      method: 'patch',
+      url: `/v1/api/game/${gameId}`,
+      timeout: 4000,    // 4 seconds timeout
+      data: {
+          team_1_turn: !this.state.team_1_turn,
+          current_round: this.state.round,
+      }
+    })
+    .then((response) => {
+        this.saveState(response.data.result[0])
     })
     .catch(function (error) {
         console.log(error);
     });
   }
 
+  componentSwitch(gameId) {
```

Comment:
I added this new switch to show different game components depending on the current round number..